scripts/hashauthpw

```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env python
 import sys
 
-from passwordhash import generate_password_digest
+from hashauthenticator import generate_password_digest
 
 if __name__ == '__main__':
-  if len(sys.argv) not in [3,4]:
-    print('Usage: hashauthenticator secret_key user [len]')
+  if len(sys.argv) not in [2,3,4]:
```

Comment:
Use `argparse` which makes this much more clear.

Reply:
Yeah, I know.